### PR TITLE
documentation 3rd iteration

### PR DESCRIPTION
**Pull request by tdeheurles**

### CODE REVIEW

File: docs/deployment/build-id.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Understand: Build ids
+
+We use a build id to define versions for *Reactive Trader Cloud*. This *build id* is a string. It could be a number or a keyword. The *build id* is first used when we *build* to name the docker images. It then can be used to run or deploy that specific reactive trader cloud version.
+
+## Defined build ids
+### CircleCi
+When [circle ci][circleci] runs a build, it generates docker images and uses the *circle ci build number* as the *build id*. Then on specific branches like *master*, CircleCi push the docker images to [docker hub][dockerhub]
```
Comment (MaciejAdaptive) [inline docs/deployment/build-id.md]: '...CircleCi pushes...'

File: docs/deployment/rtc-deployment.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+## Do the deployment
+- follow the [bash setup][bash-setup] and start a bash terminal
+- choose your [environment name][environment-name]. For this example we will use `myenv`
+- [identify your build_id][build-id]. Here we will use `buildid`
+- deploy your environment:
+    ```
+    # /!\ Bash
+    `./deploy/kubernetes/deploy myenv buildid`
+    ```
+- wait a moment and see the ressources :
```
Comment (MaciejAdaptive) [inline docs/deployment/rtc-deployment.md]: resources (single 's')

File: docs/deployment/run-minikube.md [hunk 3]
```diff
@@ -1,7 +1,39 @@
...
+    ```
+- Cleanup
+    ```powershell
+    # /!\ admin powershell
+    .\deploy\minikube\Remove-Minikube.ps1
+    ```
+
 ### Mac/Linux
-No rtc script implementation exist for now
+No rtc script implementation exist for now.
```
Comment (MaciejAdaptive) [inline docs/deployment/run-minikube.md]: exists

File: docs/deployment/understand-rtc-build-and-deployments.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ With docker installed, you can:
 
 #### Added value
 We have added the Minikube feature to rtc in order to be able to deploy locally in the same way we deploy to gcloud.  
-This let us use the same [deployment code and mechanism](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/docs/deploy/rtc-deployment.md).  
+This let us use the same [deployment code and mechanism](https://github.com/AdaptiveConsulting/ReactiveTraderCloud/blob/master/docs/deployment/rtc-deployment.md).  
 
 Docker let us run rtc locally without kubernetes but that code is custom and requires maintenance as docker is in a constant evolution with constant breaking changes (mainly for windows/mac).
```
Comment (MaciejAdaptive) [inline docs/deployment/understand-rtc-build-and-deployments.md]: Docker lets us...

File: docs/deployment/build-id.md (hunk 1 above)
Comment (tdeheurles) [reply docs/deployment/build-id.md]: done

File: docs/deployment/build-rtc-locally.md [hunk 5]
```diff
@@ -110,3 +103,12 @@ reactivetrader/web          0.0.localbuild       d0412eae6ff5        15 seconds
...
 ```
+
+[docker-setup]: ./docker-setup.md
+[build-id]: ./build-id.md
+[dotnetcore-image]: https://hub.docker.com/r/weareadaptive/dotnet/
+[crossbar-image]: https://hub.docker.com/r/weareadaptive/crossbar/
+[eventstore-image]: https://hub.docker.com/r/weareadaptive/eventstore/
+[nodejs-image]: https://hub.docker.com/r/weareadaptive/node/
+[nginx-image]: https://hub.docker.com/r/weareadaptive/nginx/
+[docker-hub]: https://hub.docker.com/u/reactivetrader/dashboard/
```
[the remark this answers is not in the document]
Comment (tdeheurles) [reply docs/deployment/build-rtc-locally.md]: Switch to store.docker.com

File: docs/deployment/run-minikube.md (hunk 3 above)
Comment (tdeheurles) [reply docs/deployment/run-minikube.md]: done

**Pull request merged**